torchvision/io/image.py

```diff
@@ -37,7 +37,8 @@ def decode_png(input):
     Returns:
         output (Tensor[image_width, image_height, 3])
     """
-    if not isinstance(input, torch.Tensor) or input.numel() == 0 or input.ndim != 1:
+    if not (isinstance(input, torch.Tensor) or
+            input.numel() == 0 or input.ndim != 1):
```

Comment:
This introduced a bug in the code, as the parenthesis is now negating the whole expression, and not only the `isinstance`